packages/react-app-scaffolder/app/index.js

```diff
@@ -85,14 +105,25 @@ module.exports = class extends Generator {
     }
   }
 
+  _addStyleSolution() {
```

Comment:
add optional sass options:
- base-is-linaria contains login style written in linaria syntax since it can't interpolate sass variable
- base-is-sass contains login style written in sass